.github/workflows/deploy-to-QA.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy To QA
+
+on:
+  workflow_call:
+
+jobs:
+  Deploy-to-QA:
+    runs-on: ubuntu-latest
+    environment: Staging
+    steps:
+      - uses: actions/checkout@v3
+      - name: Login to Github
+        env:
+          COMMIT_AUTHOR: ${{ secrets.CI_COMMIT_AUTHOR }}
+          COMMIT_EMAIL: ${{ secrets.CI_COMMIT_EMAIL }}
+        run: |
+          git config --global user.name $COMMIT_AUTHOR
+          git config --global user.email $COMMIT_EMAIL
+          
+      - name: Check if fixed version is on Jira
+        env:
+          JIRA_TOKEN: ${{ secrets.CI_JIRA_TOKEN }}
+          BRANCH_NAME: ${{github.ref_name}}
+        run: |
+          fixed_version_found=false
+          plugin_version=$(echo "$BRANCH_NAME" | grep -Eo '[0-9].[0-9].[0-9]+')
```

Comment:
will not work on latest version with 2 digits minor version.
should be '[0-9].[0-9]**+**.[0-9]+'